oeps/oep-0005/decisions/0004-To-Make-Or-Not-To-Make.rst

```diff
@@ -0,0 +1,65 @@
+To Make or Not to Make
+======================
+
+
+Status
+------
+
+Accepted
+
+
+Context
+-------
+
+For Decentralized Devstack, an common interface is necessary to enable a OpenedX Developer to add contributions to the many independent services in Open edX codebase.
+
+Options
+~~~~~~~
+
+1. The current Devstack uses make targets as an basic interface for developers.
+
+2. Use docker-compose commands directly
+
+Case for and against make targets
+`````````````````````````````````
+
+For:
+
+- This standard already exists and is known by developers at edx.
+
+  * If we don't create a make interface now, sevices might create their on make interface, which might/will result in non-standard make experience
+- make targets decrease verbosity and memorization
+
+Against:
+
+- make targets do not provide much value
+
+  * most targets wrap commands that are/can be only few characters longer
```

Comment:
> The make wrappers can provide substantially more value than just abbreviation; I've added a few above, but there may be more I didn't think of just now.

Yes, but that doesn't change the point here that `make down|pull|up|restart` is not much shorter than `docker-compose down|pull|up|restart`, but does add a level of indirection.

Reply:
This was more about the statement "make targets do not provide much value", apparently only supported by a small amount of abbreviation.

Reply:
Mm, true; my bad. "make targets do not provide much value" is a pretty strong statement.

Reply:
I changed the contentions wording from "make targets do not provide much value" to  "these make targets do not sufficiently decrease verbosity". Which I think takes care of this contention. Feel free to unresolve this in case you disagree.